README.md

```
@@ -118,35 +122,37 @@ This creates a zero-cost pointer — no duplication, files exist only once. You
+- **Compiler Flags**: We automatically inject `-xc++` and `-std=c++17` along with version-specific defines (`__MQL4__`/`__MQL5__`) to help clangd understand MQL syntax.
+- **CPU Architecture (MQL5)**: `mql_tools.Compile.CpuArchitecture` lets you compile for AVX/AVX2/AVX512 instruction sets via a temporary `.mqproj` wrapper. Only affects MQL5; incompatible binaries will fail to load on older CPUs. Market uploads require `x64` (default).
+- **Relative Paths & Portable Mode**: Settings like `mql_tools.Metaeditor.Metaeditor5Dir` and `mql_tools.Metaeditor.Include5Dir` now support `${workspaceFolder}` variable substitution and relative paths. This is perfect for portable MetaTrader installations:
+  ```json
+  {
+    "mql_tools.Metaeditor.Metaeditor5Dir": "${workspaceFolder}/../MetaEditor64.exe",
+    "mql_tools.Metaeditor.Include5Dir": "${workspaceFolder}"
+  }
+  ```
+- **Third-party Libraries**: If you use libraries like `JAson.mqh` and clangd reports "Unknown type" errors, you have two options:
+
+      **Option A** - Add to settings (global, affects all files):
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)